docs/fundamentals/flow/yaml-spec.md

```diff
@@ -0,0 +1,66 @@
+(flow-yaml-spec)=
+# YAML specification
+
+This page outlines the specification for valid Flow YAML files.
+
+Such YAML configurations can be used to generate a Flow object via `Flow.load_config('flow.yml')`.
+
+To generate a YAML configuration from a `Flow` Python object, run `f.save_config()`.
+
+## Example
+
+The following constitutes an example Flow configuration:
+
+`flow.yml`/`flow.yaml`:
+```yaml
+jtype: Flow
+version: '1'
+with:
+  protocol: http
+metas:
+  - py_modules:
+      - executor.py
+executors:
+# inline Executor YAML
+- name: firstexec
+  uses:
+    jtype: MyExec
+    py_modules:
+      - executor.py
+# reference to Executor YAML
+- name: secondexec
+  uses: indexer.yml
+  workspace: /home/johannes/.config/JetBrains/PyCharmCE2022.1/scratches/indexed
```

Comment:
what about this workspace? why is not explained? I also would suggest for u just to pass some random value here

Reply:
Below it says that here we can put any argument that `add()` would take, and links to where to find that info. This is meant just as an example of where to actually put such an arg (how to indent etc)